docs/types.propDefinition.md

```diff
@@ -133,12 +133,24 @@ propertyName: {
 propertyName: "typeName"
 ```
 
-@type {Constructor} Defines a [can-define.types.TypeConstructor Type] setting with a constructor
-function.  Constructor functions are identified with [can-util/js/types/types.isConstructor].
+@type {Constructor} Either creates a method or Defines a [can-define.types.TypeConstructor Type] setting with a constructor function.  Constructor functions are identified with [can-reflect.isConstructorLike].
 
 ```
 propertyName: Constructor
 ```
+OR
+```
+propertyName: function() {}
+```
+
+For example:
+```js
+subMap: DefineMap // <- calls constructor
```

Comment:
Maybe this should say
> sets `Type` to `DefineMap`

This seems to imply that the constructor is called immediately, which it doesn't.